src/components/avatar-builder/EmojisListHeader.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { StyleSheet, Text } from 'react-native';
+import Animated, { useAnimatedStyle } from 'react-native-reanimated';
+import { fonts } from '@rainbow-me/styles';
+import { useTheme } from '@rainbow-me/theme';
+import { magicMemo } from '@rainbow-me/utils';
+
+interface Props {
+  title: string;
+  showSectionTitles?: boolean;
+  nextCategoryOffset: Animated.SharedValue<number>;
+}
+
+const EmojisListHeader = ({
+  title,
+  showSectionTitles,
+  nextCategoryOffset,
+}: Props) => {
+  const { colors } = useTheme();
+
+  const animatedStyle = useAnimatedStyle(() => ({
+    opacity: nextCategoryOffset.value,
+  }));
+
+  if (showSectionTitles) {
+    return (
+      <Animated.View
+        style={[
+          sx.sectionHeaderWrap,
+          { backgroundColor: colors.white },
+          animatedStyle,
+        ]}
+      >
+        <Text
```

Comment:
Shouldn't we use text from design system? Or is this one looking different enough to leave no other option here?

Reply:
Honestly, I left it as it is. I could rewrite it a little bit, but it will definitely take some time and I am not sure we want to do it.